lua/apisix/schema_def.lua

```diff
@@ -276,7 +276,7 @@ local upstream_schema = {
         enable_websocket = {
             description = "enable websocket for request",
             type        = "boolean",
-            default     = false
+            default     = nil
```

Comment:
we can delete this line, it is useless ^_^

Reply:
ok